src/System.Diagnostics.Process/tests/ProcessTests.Unix.cs

```diff
@@ -72,6 +72,25 @@ public void ProcessStart_UseShellExecute_OnLinux_ThrowsIfNoProgramInstalled()
             }
         }
 
+        [Fact]
+        [OuterLoop("Opens program")]
+        [PlatformSpecific(TestPlatforms.Linux)]
+        public void ProcessStart_DirectoryNameInCurDirectorySameAsFileNameInExecDirectory_Success()
+        {
+            string fileToOpen = "dotnet";
+            var dotnetFolderInCurDirectory = Path.Combine(Environment.CurrentDirectory, "dotnet");
+            if (!Directory.Exists(dotnetFolderInCurDirectory))
+            {
+                Directory.CreateDirectory(dotnetFolderInCurDirectory);
+            }
+            using (var px = Process.Start(fileToOpen))
+            {
+                px.Kill();
```

Comment:
I would add `Assert.IsNotNull(px);` even though that's redundant, to illustrate what the test is verifying.